README.md

```diff
@@ -12,6 +12,19 @@ A web application for providing access to Digital Objects in SDR Storage.
   + oracle - integration, staging, production
 
 ## Getting Started
+
+- `git clone` and `cd` into the cloned repository
+- setup mysql
+
+  ```sh
+  echo "For db creation, enter the 'root' user password:"
+  mysql --user=root -p < db/mysql_db_create.sql
```

Comment:
Anyone running this stuff is most likely familar with SQL and can read the .sql file to find that out, as you probably did already.  Why add extra commentary unless you want it to be in an echo?